web/src/compose_ui.js

```diff
@@ -2,12 +2,20 @@ import autosize from "autosize";
 import $ from "jquery";
 import {insert, replace, set, wrapSelection} from "text-field-edit";
 
+import render_compose_stream_dropdown_privacy_icons from "../templates/compose_stream_dropdown_privacy_icons.hbs";
+
 import * as common from "./common";
+import * as compose from "./compose";
+import * as compose_actions from "./compose_actions";
+import * as compose_state from "./compose_state";
```

Comment:
I suspect these new imports are going to generate import cycles as well, hmm. I think the right plan is probably for all your new code here to just be a new module `compose_recipient.js`. We can put both these functions and also the new functions you'd added to `compose_actions`, like `export function close_compose_stream_dropup`, in it; I think we'll be a lot happier with that module structure for other reasons too.

Reply:
That sounds like a great solution, thanks!

Reply:
Done and pushed